src/components/UserList/types.ts

```diff
@@ -0,0 +1,17 @@
+export type filterType = {
```

Comment:
```suggestion
export type FilterType = {
```

Tên type nên để kiểu UpperCamelCase